### PR TITLE
Move test fixtures from libs to contracts

### PR DESCRIPTION
**Pull request by pcppcp**
- moves contract fixtures from `raiden-libs` to `raiden-contracts`
- also fixes fixture names
- do not merge until other repos are ready for the change!

### CODE REVIEW

File: raiden_contracts/tests/fixtures/contracts.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import logging
+from web3.utils.events import get_event_data
+from eth_utils import is_address
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def contracts_manager():
+    """Reimplement this - fixture should return raiden-lib's contract manager.
```
Comment (loredanacirstea) [inline raiden_contracts/tests/fixtures/contracts.py]: We already have the `contracts_manager` fixture in https://github.com/raiden-network/raiden-contracts/blob/a17d6d908fa85d9cf963f81e8fc0e7a76aaa33fb/raiden_contracts/tests/fixtures/contract_manager.py#L6, this one should be removed.
`raiden-lib's contract manager` - the contract manager should only stay in this repo.

File: raiden_contracts/tests/fixtures/contracts.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        (return ContractManager(CONTRACTS_SOURCE_DIRS))"""
+    raise NotImplementedError(
+        'In order to use contract fixtures please override contracts_manager fixture.'
+    )
+
+
+@pytest.fixture
+def contract_deployer_address(faucet_address) -> str:
```
Comment (loredanacirstea) [inline raiden_contracts/tests/fixtures/contracts.py]: We have this defined in https://github.com/raiden-network/raiden-contracts/blob/a17d6d908fa85d9cf963f81e8fc0e7a76aaa33fb/raiden_contracts/tests/conftest.py#L8, should be moved here.

File: raiden_contracts/tests/test_token_network.py [hunk 3]
```diff
@@ -7,19 +7,25 @@ def test_version(token_network):
     assert token_network.call().contract_version()[:2] == raiden_contracts_version[:2]
 
 
-def test_constructor_call(web3, get_token_network, custom_token, secret_registry, get_accounts):
+def test_constructor_call(
+    web3,
+    get_token_network,
+    custom_token,
+    secret_registry_contract,
+    get_accounts
```
Comment (loredanacirstea) [inline raiden_contracts/tests/test_token_network.py]: Code style - add another 4 spaces to args.

File: raiden_contracts/tests/test_token_network_registry.py [hunk 4]
```diff
@@ -10,11 +10,17 @@
...
+    assert (token_network_registry_contract.call().contract_version()[:2]
+            == raiden_contracts_version[:2])
 
 
-def test_constructor_call(web3, get_token_network_registry, secret_registry, get_accounts):
+def test_constructor_call(
+    web3,
+    get_token_network_registry,
+    secret_registry_contract,
+    get_accounts
```
Comment (loredanacirstea) [inline raiden_contracts/tests/test_token_network_registry.py]: code style

File: raiden_contracts/tests/test_token_network_registry.py [hunk 5]
```diff
@@ -28,74 +34,83 @@ def test_constructor_call(web3, get_token_network_registry, secret_registry, get
...
+    registry = get_token_network_registry([secret_registry_contract.address, chain_id])
+    assert secret_registry_contract.address == registry.call().secret_registry_address()
     assert chain_id == registry.call().chain_id()
 
 
-def test_create_erc20_token_network_call(token_network_registry, custom_token, get_accounts):
+def test_create_erc20_token_network_call(
+    token_network_registry_contract,
+    custom_token,
+    get_accounts
```
Comment (loredanacirstea) [inline raiden_contracts/tests/test_token_network_registry.py]: code style

File: raiden_contracts/tests/test_token_network_registry.py [hunk 6]
```diff
@@ -28,74 +34,83 @@ def test_constructor_call(web3, get_token_network_registry, secret_registry, get
...
+    assert token_network.call().chain_id() == token_network_registry_contract.call().chain_id()
 
 
-def test_events(register_token_network, token_network_registry, custom_token, event_handler):
-    ev_handler = event_handler(token_network_registry)
+def test_events(
+    register_token_network,
+    token_network_registry_contract,
+    custom_token,
+    event_handler
```
Comment (loredanacirstea) [inline raiden_contracts/tests/test_token_network_registry.py]: code style

File: raiden_contracts/tests/fixtures/contracts.py (hunk 2 above)
Comment (pcppcp) [reply raiden_contracts/tests/fixtures/contracts.py]: this is a project-specific variable, each project should redefine this fixture (I'll probably move it to libs)

**Pull request merged**